src/commands/showReferrer.ts

```diff
@@ -0,0 +1,87 @@
+import * as vscode from 'vscode';
+import { execAsync } from '../support/execAsync';
+import { window } from 'vscode';
+import * as os from 'os';
+import * as path from 'path';
+import { access } from 'fs/promises'; 
+
+const errorMessage = 'ORAS not found on default path. Download ORAS or update path: ';
+const goToOrasButton = 'Download ORAS';
+const laterButton = 'Later';
+const validTag = /^[a-z0-9./_:-]+$/; 
+
+// check oras exist on user computer
+async function getOrasPath(path: any, bool : boolean): Promise<[string, boolean]> {
+    const isWindows = os.platform() === 'win32'; 
+    const homedir = os.homedir(); 
+    const orasExecutable = isWindows ? 'oras.exe' : 'oras'; 
+    const dirOras = path.join(homedir, 'bin', 'oras', orasExecutable);
+
+        try {
+            await access(dirOras);
+            return [dirOras , true];
+        } catch {
+            vscode.window.showInformationMessage(errorMessage, goToOrasButton, laterButton)
+            .then(selection => {
+                if (selection === goToOrasButton) {
+                    vscode.env.openExternal(vscode.Uri.parse('https://oras.land/docs/installation'));
+                } else if (selection === laterButton){
```

Comment:
Do we need this else if? It doesn't seem to do anything?

Reply:
I added it for the user experience, it can be taken out

Reply:
There is no concern with having the button itself. Does the if statement here affect its functionality though?

Reply:
The else if doesn't change the functionality of the goToOrasButton which takes the user to the official Oras installation page.

Reply:
Then lets remove it. You shouldn't have empty branches on if statements.